global/prometheus-alertmanager-operated/Chart.yaml

```
@@ -2,13 +2,13 @@ apiVersion: v2
 type: application
 description: Prometheus Alertmanager via operator.
 name: prometheus-alertmanager-operated
-version: 4.6.4
+version: 4.6.5
```

```suggestion
version: 4.7.1
```

This PR is going to be merged first and introduces `4.7.0`

https://github.com/sapcc/helm-charts/pull/9453/